src/style.css

```diff
@@ -1,3 +1,101 @@
 body {
   margin: 0;
+  font-size: 12px;
+
+  --select-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  font-family: Roboto, Arial, sans-serif;
+  background-color: white;
+}
+
+.logo__link {
+  display: inline-flex;
+  margin-top: auto;
+  margin-bottom: auto;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.header__menu {
+  margin-right: 50px;
+}
+
+.header__list {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.header__item {
+  display: block;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+.header__item:nth-child(-n + 7) {
+  margin-right: 20px;
+}
+
+.header__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  height: 100%;
+  height: 60px;
+  line-height: 60px;
+  font-weight: 500;
+  margin: 0;
+  padding: 0;
+  display: block;
+}
+
+.is-active {
+  color: var(--select-color);
+}
+
+.is-active::after {
+  position: absolute;
+  content: '';
+  display: block;
+  background-color: var(--select-color);
+  height: 4px;
+  width: 37px;
+  border-radius: 8px;
+  top: 56px;
```

Comment:
To not add specific px width, you can set position: relative for .is-active element and addwidth: 100%for after element

Also better to add bottom: 0 instead of setting top